Eradication/Assortivity.cpp

```diff
@@ -453,32 +465,19 @@ namespace Kernel
                 p_partner_B->SetAssortivityIndex(m_RelType, b_index);
```

Comment:
I realize this code hasn't changed, but I guess I'm wondering what we're using the SetAssortivityIndex for? we're using a_index and b_index directly below. Is this something that's used elsewhere and it's just a convenient place to set it for later?

Reply:
I see that we just get the index instead of running it through the function every time. Is this a time saving? Do folks without IP not have an assortivity index? We run them through the int a_index = func(this, pPartnerA); every time which feels really close to this: a_index = GetIndex(func(this, pPartnerA)); Just make func1(this, pPartnerA) = GetIndex(func(this, pPartnerA)) and we can do the a_index = func(this, pPartnerA);? (I am just curious, I realize this isn't actually part of the review)